site/src/routes/embedding-knn/AgreementClusterStaticPlot.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let data: any[] = [];
+
+  var names = data.map((d) => d["ego_primary_label"]);
+  names = names.slice(0, 20);
+  var selected: string = "normoc";
+</script>
+
+<div class="selection">
+  <b>Species:</b>
+  {#each names as name}
+    <label>
+      <input type="radio" bind:group={selected} name="names" value={name} />
```

Comment:
```suggestion
      <input type="radio" bind:group={selected} name={name} value={name} />
```

otherwise each of the inputs will have the same name